infrastructure/aws-accounts/keys/christos.loverdos.base64

```diff
@@ -0,0 +1,40 @@
+mQINBFopSEMBEADEjFgKRTfrOtadRouo9FFmDGVCReZ1k3n4X3sSHFqCjQ5Kgb+QfngdQA/iL1eU
```

Comment:
Do we want Christos' key (similarly for others)? Also some of the keys are empty, which seems suspicious.

Reply:
yes because we are managing the mantis account which has more than plutus stuff in it. The reason I want to manage it from the plutus repo is that mantis stuff is deprecated/dying so it should be possible to remove some time in 2021 and then the account will be plutus only. Additionally I want to use it (already have actually) to add new plutus devs to the AWS account.

Reply:
> we are managing the mantis account... from the plutus repo

:scream: 

I'm going to trust that you know what you're doing, but this sounds a little scary...

Reply:
By "the mantis account" I mean just the name. It happened to be the mantis account and that was inherited by us. Day-to-day it is only used by Plutus but mantis-iele-ops hasn't disappeared completely yet.

Reply:
another way to look at this is that I want to make sure a team has ownership of this AWS account, at the moment the only permanent team that uses it is the plutus team.